materialbarcodescanner/src/main/java/com/edwardvanraak/materialbarcodescanner/BarcodeGraphic.java

```diff
@@ -97,6 +96,7 @@ public void draw(Canvas canvas) {
         /**
          * Draw the top left corner
          */
+        int mCornerWidth = 64;
```

Comment:
Seems to be. Looks like this class is supposed to draw a barcode when it's scanned, but I don't think this code is executed when scanning the pdf417...